### PR TITLE
fix(builds): Pulls in unopinionated spinnaker-gradle-project plugin

### PR DESCRIPTION
**Pull request by ttomsu**
Do not merge with release candidate libraries - needs https://github.com/spinnaker/spinnaker-gradle-project/pull/47 to be released.

Supercedes https://github.com/spinnaker/kork/pull/157

Ref: https://github.com/spinnaker/spinnaker/issues/2844

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -57,4 +57,11 @@ allprojects {
   spinnaker {
     dependenciesVersion = spinnakerDependenciesVersion
   }
+
+  //c&p this because NetflixOss reverts it to 1.7 and ends up getting applied last..
+  project.plugins.withType(JavaBasePlugin) {
```
Comment (dibyom) [inline build.gradle]: Does this have to be added to each project now?

File: gradle/buildViaTravis.sh [hunk 2]
```diff
@@ -1,24 +1,27 @@
 #!/bin/bash
 # This script will build the project.
 
+GRADLE="./gradlew -I gradle/init-publish.gradle --no-daemon --max-workers=1"
```
Comment (dibyom) [inline gradle/buildViaTravis.sh]: Should we be hardcode `max-workers` to 1? The default seems to be the number of CPUs

File: kork-web/src/main/groovy/com/netflix/spinnaker/config/CompositeBuildTest.groovy [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.netflix.spinnaker.config
+
+import groovy.util.logging.Slf4j
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
```
Comment (dibyom) [inline kork-web/src/main/groovy/com/netflix/spinnaker/config/CompositeBuildTest.groovy]: Is this file meant to be checked in?

File: build.gradle (hunk 1 above)
Comment (ttomsu) [reply build.gradle]: I'm actually hoping Cam can point me to where I need to fix this in the plugin to not need the c&p

File: gradle/buildViaTravis.sh (hunk 2 above)
Comment (ttomsu) [reply gradle/buildViaTravis.sh]: For OSS Travis CI builds, I don't think you get more than 1, but I'm not 100% sure.

File: kork-web/src/main/groovy/com/netflix/spinnaker/config/CompositeBuildTest.groovy (hunk 3 above)
Comment (ttomsu) [reply kork-web/src/main/groovy/com/netflix/spinnaker/config/CompositeBuildTest.groovy]: How would you _know_ if your super special Kork change worked?!

**Comment by ttomsu** [comment]
Going to merge this and release as an RC to test that publishing still works.

**Pull request merged**